packages/components/src/unit-control/utils.js

```diff
@@ -11,15 +11,15 @@ import { Platform } from '@wordpress/element';
 
 const isWeb = Platform.OS === 'web';
 
-const allUnits = {
+export const allUnits = {
```

Comment:
What is the reason for exporting `allUnits`? It doesn't seem to be required in any other file

Reply:
In an earlier version of my tests, I grabbed `allUnits`, but I since updated the tests not to require it, so have reverted this change. Thanks for picking that up!